bin/extract-ORFs.py

```diff
@@ -46,13 +46,35 @@
         help='Only ORFs of a maximum of this length will be written to'
         'stdout.')
 
+    group = parser.add_mutually_exclusive_group()
+
+    group.add_argument(
+        '--kozakInfo', default=False, action='store_true',
+        help='A file with Kozak consensus information will be written'
```

Comment:
I usually write these long help strings with `help=('.....')` so that my editor indents the 2nd, 3rd etc lines to the same level as the first. Not important...

Reply:
Probably I would get rid of this flag and just write the Kozak info if `--kozakInfoFile` is given.

Reply:
You need a space at the end of each line (I mean in the Python file) of your multi-line help strings.

Reply:
> I usually write these long help strings with `help=('.....')` so that my editor indents the 2nd, 3rd etc lines to the same level as the first. Not important...

done

Reply:
> Probably I would get rid of this flag and just write the Kozak info if `--kozakInfoFile` is given.

done

Reply:
> You need a space at the end of each line (I mean in the Python file) of your multi-line help strings.

done